src/main/java/edu/ucsb/cs156/rec/controllers/RequestTypesController.java

```diff
@@ -101,11 +101,11 @@ public RequestType postRequestType(
     }
 
     /**
-     * Update a single requestType
+     * Update a single request type
      * 
-     * @param id         id of the requestType to update
-     * @param incoming   the new RequestType
-     * @return           the saved requestType
+     * @param id  the name of the request type 
```

Comment:
No, you had it right the first time.  It's the id of the request type. Note that it's a Long value.
```suggestion
     * @param id  the id of the request type 
```

Reply:
Fixed